src/cmake.mk

```diff
@@ -3,8 +3,8 @@
 PKG             := cmake
 $(PKG)_WEBSITE  := https://www.cmake.org/
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 3.10.0
-$(PKG)_CHECKSUM := b3345c17609ea0f039960ef470aa099de9942135990930a57c14575aae884987
+$(PKG)_VERSION  := 3.11.4
```

Comment:
There's a later cmake 3.12.1 - any reason not to use that?

Reply:
Yes, cmake in boost fails.
Could not find the following Boost libraries: boost_context

Reply:
Okay, I'd suggest leaving cmake out of this PR in that case - later boost has [other issues](https://github.com/mxe/mxe/pull/2043#issuecomment-414269871) that the newest cmake may help with.